lib/sync/dataHandlers.js

```diff
@@ -259,6 +261,16 @@ module.exports = function(options) {
       handler(datasetId, dataHash, metaData, callback);
     },
 
+    /**
+     * @param datasetId
+     * @param metaData
+     * @param callback
+     */
+    countCollisions: function(datasetId, metaData, callback) {
```

Comment:
@aidenkeating a collision is generated from the pending processor, for example: https://github.com/feedhenry/fh-mbaas-api/blob/FH-3123-sync-horizontal-scaling/lib/sync/pending-processor.js#L39.

We can just increment the collision counter at this point, if we keep it on the dataset client document. This way we don't need to count the collisions at all.